src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1391,5 +1391,9 @@ internal enum ErrorCode
         ERR_TupleExplicitNamesOnAllMembersOrNone = 8204,
 
         ERR_PredefinedTypeMemberNotFoundInAssembly = 8205,
+        ERR_MissingDeconstruct = 8206,
```

Comment:
The tuple errors are far out of numeric sequence, starting at 8200. Please move them into lexical order that corresponds to their numeric values.